templates/cluster-template.yaml

```diff
@@ -130,4 +136,6 @@ spec:
       joinConfiguration:
         nodeRegistration:
           kubeletExtraArgs:
+            cloud-provider: external
+            provider-id: ibmvpc://${CLUSTER_NAME}/'{{ v1.local_hostname }}'
```

Comment:
By setting this `provider-id` I saw that my machines and the machinedeployment itself reached the `Running` status.
Prior to that, they got stuck on the `Unavailable` state, even though they were Ready and receiving pods.

Reply:
I think this id should be obtained from cloud ? e.g a VM/machine is created and a unique ID 
for it, then use that ID for the provider-id?